pylab/website/tests/test_voting.py

```diff
@@ -9,16 +9,16 @@
 
 class VotingTests(WebTest):
 
-    def test_voting_page(self):
+    def test_voting_and_voting_poll_details_page(self):
         u1 = User.objects.create(username='u1')
+        u2 = User.objects.create(username='u2')
 
         p1 = Project.objects.create(author=u1, title='Test title 1', description='Test description')
         p2 = Project.objects.create(author=u1, title='Test title 2', description='Test description')
 
         vp = VotingPoll.objects.create(author=u1, title='Test voting poll', description='Test description')
```

Comment:
Use factory boy factories instead of raw Django models. When data structure changes it is much easier to maintain tests, when all fixtures are created using factory boy factories.